pwndbg/commands/heap.py

```diff
@@ -96,6 +98,29 @@ def arena(addr=None):
 
     print(main_arena)
 
+
+@pwndbg.commands.ParsedCommand
+@pwndbg.commands.OnlyWhenRunning
+def arenas():
+
+    heap  = pwndbg.heap.current
+    addr  = None
+    arena = heap.get_arena(addr)
+    main_arena_addr = int(arena.address)
+    fmt = '[%%%ds]' % (pwndbg.arch.ptrsize *2)
+    while addr != main_arena_addr:
+        
+        h = heap.get_region(addr)
+        if not h:
+            print(red('Could not find the heap'))
+            return
+        
+        print(fmt%(hex(addr) if addr else 'main'),str(h))
```

Comment:
Just `h` instead of `str(h)` - print calls `obj.__str__` automagically. Also more spaces ;P